util/src/main/scala/Money.scala

```diff
@@ -248,34 +249,49 @@ object Money {
   val FractionDigitsField: String = "fractionDigits"
   val TypeName: String = "centPrecision"
 
-  def fromDecimalAmount(amount: BigDecimal, currency: Currency)(implicit mode: RoundingMode) =
+  def fromDecimalAmount(amount: BigDecimal, currency: Currency)(implicit mode: RoundingMode): Money =
     Money(amount.setScale(currency.getDefaultFractionDigits, mode), currency)
 
-  val bdOne = BigDecimal(1)
-  val bdTen = BigDecimal(10)
+  val bdOne: BigDecimal = BigDecimal(1)
+  val bdTen: BigDecimal = BigDecimal(10)
 
   private val centFactorZeroFractionDigit = bdOne / bdTen.pow(0)
   private val centFactorOneFractionDigit = bdOne / bdTen.pow(1)
   private val centFactorTwoFractionDigit = bdOne / bdTen.pow(2)
   private val centFactorThreeFractionDigit = bdOne / bdTen.pow(3)
+  private val centFactorFourFractionDigit = bdOne / bdTen.pow(4)
+
+  def cachedCentFactor(currencyFractionDigits: Int): BigDecimal =
```

Comment:
do we want to be part of the API of `Money`? Or can it be private?

Reply:
Good catch 👍 
Let's keep the magic private https://github.com/sphereio/sphere-scala-libs/pull/179/commits/d2fdf63558bf96926cc5b0971349e729c399d935